tests/src/Result/ValidationResultCliRendererTest.php

```
@@ -190,17 +192,194 @@
+        // Test verbose mode (no prefix)
+        $result = $method->invoke($this->renderer, $validator, $issue, 'TestValidator', true);
+        $this->assertStringNotContainsString('(TestValidator)', $result);
     }
 
     private function createMockValidator(): ValidatorInterface|MockObject
     {
         $validator = $this->createMock(ValidatorInterface::class);
-        $validator->method('renderIssueSets');
-        $validator->method('explain')->willReturn('Mock validator explanation');
+        $validator->method('resultTypeOnValidationFailure')->willReturn(ResultType::ERROR);
+        $validator->method('formatIssueMessage')->willReturnCallback(fn (Issue $issue, string $prefix = '', bool $isVerbose = false): string => "- <fg=red>ERROR</> {$prefix}Validation error");
```

PHPMD (2.15.0)

Avoid unused parameters such as '$isVerbose'. (Unused Code Rules)

(UnusedFormalParameter)